### ISSUE TITLE
NewRedisClient has no REDIS access password input parameter when redis-server --requirepass security is enabled

### ISSUE THREAD

**Issue by lidernetworktecnologia** [danieleteti/delphiredisclient#36]
I would like this input parameter to be implemented as it already exists for host and port. This is because when publishing REDIS in the cloud, it is necessary to activate the --requirepass security in the redis.conf file to prevent unauthorized access. I would like to congratulate you for the component, it is excellent, but this input parameter is missing!

**Comment by Zhou171542036** [comment]
来信已收到。

**Comment by danieleteti** [comment]
Thanks for your kind words.
Did you check the AUTH method?

DT

Il ven 9 giu 2023, 14:17 LIDERNetwork Tecnologia by Junior LIDER <
***@***.***> ha scritto:

**Comment by lidernetworktecnologia** [comment]
Yes, I tried this method but as new does not have the parameter it is created without the password and therefore cannot connect. The AUTH method only works after the object is created, so you would need to pass this parameter before new, which is currently not possible
